Documentation/upgrade.md

```diff
@@ -59,6 +60,16 @@ spec:
       journalSizeMB: 1024
 ```
 
+### Specify a MonCount if not set
+It is required to set the `monCount` in the Cluster CRD, before continuing.
+The `monCount` must be set to the count of mons you want to have.
+If it wasn't specified yet, the following cases apply for the automatically calculated `monCount`:
+* 1x Node = 1 Mon
+* 2x Nodes = 1 Mon (Don't use `2` here)
+* 3x Nodes and more = 3 Mons
+
+To edit the Cluster CRD and set the `monCount`, you can use `kubectl edit -n rook cluster rook`.
```

Comment:
instead of requiring them to edit it before the upgrade, could we set it for them automatically during the crd conversion? We already have conversion code around this, right?

Reply:
So if the `monCount` is zero (e.g. unset), we set it to `3` here https://github.com/rook/rook/pull/1780/files#diff-ddc0fe06b6e42c6debef20293e1ee91fR92 or just leave it at `0` so it will be "overriden" here https://github.com/rook/rook/pull/1780/files#diff-8a603243c4e06abaf58c1cbd1426cf25R187 ?

Reply:
During upgrade if not set, I'd recommend we set it to 3 and set the `allowmultiplePerNode` to `true`. This will preserve the pre-upgrade behavior and reduce complexity of the upgrade.